test/src/module/command/helpTest.c

```diff
@@ -639,6 +637,61 @@ testRun(void)
+            "is more memory efficient to limit the amount of memory used for uploads.\n"                                           \
+            "\n"                                                                                                                   \
+            "A larger chunk size will generally lead to better performance because it will\n"                                      \
+            "minimize upload requests and allow more files to be uploaded in a single\n"                                           \
+            "request rather than in chunks. The disadvantage is that memory usage will be\n"                                       \
+            "higher and because the chunk buffer must be allocated per process, larger\n"                                          \
+            "process-max values will lead to more memory being consumed overall.\n"                                                \
+            "\n"                                                                                                                   \
+            "Default chunk sizes by repo type:\n"                                                                                  \
+            "\n"                                                                                                                   \
+            "* azure - 4MiB\n"                                                                                                     \
+            "* gcs - 4MiB\n"                                                                                                       \
+            "* s3 - 5MiB\n"                                                                                                        \
+            "\n"                                                                                                                   \
+            "Note that valid chunk sizes vary by storage type and by platform. For example,\n"                                     \
+            "AWS S3 has a minimum chunk size of 5MiB but S3 clones may accept lower values.\n"                                     \
+            "Terminology for chunk size varies by storage type, so when searching min/max\n"                                       \
+            "values use \"part size\" for AWS S3, \"chunk size\" for GCS, and \"block size\" for\n"                                \
+            "Azure. No attempt is made to validate configured chunk sizes so selecting an\n"                                       \
+            "invalid value will lead to errors from the storage service or undefined\n"                                            \
+            "behavior.\n"
+
+        optionHelp = zNewFmt(
+            HELP_OPTION_CHUNK
+            "\n"
+            "default:\n"
+            "  repo1: 5242880\n"
+            "  repo2: 4194304\n",
```

Comment:
So this is the meat of what we're actually testing in this test; otherwise, the rest of it is just prepping the expected output test so it can do (more-or-less) a strcmp to verify that we get the exact help output for this command given the later option setup, yes?